modules/_labs/header/react/lib/Header.tsx

```diff
@@ -129,7 +129,7 @@ const navStyle = ({themeColor}: Pick<HeaderProps, 'themeColor'>) => {
           height: 'inherit',
           transition: `color 150ms ease-out 0s`,
           cursor: 'pointer',
-          '&:first-child > *': {
+          '&:first-of-type > *': {
```

Comment:
What does this change have to do with changing focus trap libraries? Is it a necessary change?

Reply:
@NicholasBoll you are right its not related to that PR, https://github.com/storybookjs/storybook/issues/6998
I am reverting this